pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2076,6 +2076,15 @@ var _ = Describe("Converter", func() {
 		)
 	})
 
+	It("should not include serial entry in sysinfo when firmware.serial is not set", func() {
+		vmi := libvmi.New()
+		v1.SetObjectDefaults_VirtualMachineInstance(vmi)
+		domain := vmiToDomain(vmi, &ConverterContext{Architecture: archconverter.NewConverter(runtime.GOARCH)})
```

Comment:
@fossedihelm hey, is there a convention in this suite to use AllowEmulation? otherwise, this would fail if run through make test and a non privileged dev container

Reply:
I had to pass `AllowEmulation:true` to let this pass in goverall.
Too bad I did not get to resolve https://github.com/kubevirt/kubevirt/issues/13837

Reply:
> @fossedihelm hey, is there a convention in this suite to use AllowEmulation? otherwise, this would fail if run through make test and a non privileged dev container

I missed this... :man_facepalming: 
Yes, currently we need to pass it.
This is something we should improve maybe